packages/generate-design-tokens/index.js

```diff
@@ -1,20 +1,62 @@
 const fs = require('fs');
 const path = require('path');
 
-const DESIGN_TOKENS_FOLDER = path.join(__dirname, '..', '..', '.github', 'mistica-design', 'tokens');
+/*
+To run this script locally:
+
+1. Clone this repo:
+    https://github.com/Telefonica/mistica-design
```

Comment:
should we clarify that the repo will be cloned in .github folder by default if no env var is set?

Reply:
The idea of this env var is to not clone it inside .github